backend/src/controller/api.py

```diff
@@ -1,36 +1,44 @@
 from flask import Blueprint, request, current_app
 
 from models.address import AddressModel
-from models.user import UserModel
+from service.user.user_service import UserService
 
 api = Blueprint("api", __name__, url_prefix="/api/v1")
+user_service: UserService = None
 
 
-@api.route("/user", methods=["POST", "GET"])
+@api.route("/user", methods=["POST", "GET", "DELETE", "PUT"])
 def create_user():
+    """
+    User management for User CRUD
+    """
+    # TODO call get users by id , name, email (confirm)
+    if request.method == "GET":
```

Comment:
We're thinking of following RESTful route structures. Eg.
```
CREATE => POST /users
ALL RECORDS => GET /users
SINGLE RECORD => GET /users/:id
UPDATE => PUT /users/1       (with json params)
DELETE => DELETE /users/1
```
Can you separate each action as a route according to above example please?